recipes/watchdog-wms/run_test.sh

```diff
@@ -7,7 +7,7 @@ VERSION="${PKG_NAME}-${PKG_VERSION}-${PKG_BUILDNUM}"
 OUT="${PREFIX}/share/${VERSION}"
 PATH="$PATH:$OUT/core_lib/wrapper/"
 
-function timeoutbash() { bash -c '(sleep 10; kill -9 $$ 2>/dev/null) & exec $@' $0 $@; }
+function timeoutbash() { bash -c '(sleep 180; kill -9 $$ 2>/dev/null) & exec $@' $0 $@; }
```

Comment:
Since you added the `coreutils` run requirement, you can use the provided `timeout` command. This should have the advantage that it doesn't wait for the timeout even if the callee is already terminated.
Also, IIRC, `coreutils` provides GNU sed, hence you can just use `sed -i` instead of your custom `sedinline`.

Reply:
Good point, I will change that the next time I update the package.